homeassistant/components/blueprint/models.py

```diff
@@ -283,6 +283,12 @@ async def async_inputs_from_config(
 
         bp_conf = config_with_blueprint[CONF_USE_BLUEPRINT]
         blueprint = await self.async_get_blueprint(bp_conf[CONF_PATH])
+        if blueprint is None:
+            raise FailedToLoad(
```

Comment:
Maybe we should raise this error from inside `self.async_get_blueprint`. Then the return value could remain non optional.

Reply:
Done.